docs/content/latest/yugabyte-platform/create-deployments/create-universe-multi-region.md

```diff
@@ -113,28 +101,13 @@ $ java -jar /home/yugabyte/tserver/java/yb-sample-apps.jar \
             --with_local_dc <REGION>
 ```
 
-You can find the region codes for each of the nodes by browsing to the **Nodes** tab for this universe in the Yugabyte Platform console. A screenshot is shown below. In this example, the value for `<REGION>` is:
+You can find the region code of each node by navigating to **Nodes** and looking under the **CLOUD INFO** column: the first part before the slash indicates the region. For example, `us-west1`. 
```

Comment:
worth mentioning this would be back in platform UI?

Reply:
I think it's okay as is